<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePageClass</class>
 <widget class="QWidget" name="ProfilePageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProfilePage</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Profile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile</string>
   </property>
   <widget class="QLabel" name="label_Username">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ScoreInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_CoinsInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>78</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Score">
       <property name="text">
        <string>All time Score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Coins">
       <property name="text">
        <string>             Coins:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
